content/docs/hooks-recipes.md

```diff
@@ -10,3 +10,157 @@ prev: hooks-faq.html
 This is a collection of recipes that may help you start "thinking in Hooks". Feel free to play with them or copy and paste them into your project or tweak them to your needs.
 
 ## Data Fetching
+
+## Subscriptions
+
+Sometimes React components need to subscribe to an external data source. The `useSubscription` hook below shows how to do this in a way that is compatibile with React's upcoming concurrent rendering mode.
+
+> Note
+>
+> Subscriptions may not be the best solution for your use case. For example:
+> * Redux/Flux stores should use the [context API](/docs/context.html) instead.
```

Comment:
We've gotten flak for this recommendation because it led to React Redux v6 rewrite which has to back out of context now (not fast enough for some extreme use cases). I'd omit or rephrase this.

Reply:
Huh. Fair enough, although that's still in our `create-subscription` guidelines: https://github.com/facebook/react/tree/master/packages/create-subscription#when-should-you-not-use-this

Reply:
Yes, and we should have fixed it there too. There's a task but nobody got to it yet. :-)